src/resource/types.ts

```diff
@@ -24,6 +24,7 @@ export type UseWatchResourceOptions = {
   useSuspense?: boolean;
   autoRefresh?: DurationLikeObject;
   refreshOnWindowFocus?: boolean;
+  refreshOnVisibilityChange?: boolean;
```

Comment:
Wäre eher bei diesem Wording. Man könnte sonst denken es geht um die Visibility des React-Elementes, indem der Hook getriggered wurde.

```suggestion
  refreshOnDocumentVisibilityChange?: boolean;
```